tft.py

```diff
@@ -402,16 +403,14 @@ def exit_now_conditional() -> bool:
     return not league_game_already_running()
 
 
-def check_if_game_complete(wait_for_exit_buttons: bool = False) -> bool:
-    """Check if the League game is complete.
+def check_screen_for_exit_button() -> bool:
+    """
+    Checks the screen for any exit buttons we expect to appear when the player dies.
 
     Returns:
-        bool: True if any scenario in which the game is not active, False otherwise.
-    """
-    if wait_for_exit_buttons:
-        logger.info("Waiting an additional ~25s for any exit buttons")
-        time.sleep(25)
+        True if any known exit buttons were found, False if not.
 
+    """
     if onscreen(CONSTANTS["client"]["death"]):
         logger.info("Death detected")
         click_to_middle(CONSTANTS["client"]["death"])

```

Comment:
Oh -- does the [death icon](https://github.com/Kyrluckechuck/TFT-Bot/blob/main/captures/death.png) actually still show up, or should this instead be checking for the `Exit Now` button (and it's variants)?

I hadn't realized this old code was still checking for this.

Reply:
I have never seen that image in my life ever, so it probably is not in the game anymore. I can remove it :)